bin/index.js

```diff
@@ -35,17 +35,17 @@ program
   .arguments('<command>')
   .action((cmd) => {
     program.outputHelp()
-    console.log(`  ` + chalk.red(`Unknown command ${chalk.yellow(cmd)}.`))
-    console.log()
-    suggestCommands(cmd)
+    console.log(`  ` + chalk.red(`Unknown command ${chalk.yellow(cmd)}.`));
```

Comment:
I think we're going without semi-colons on the rest of the project, so it might be a good idea to remove them

Reply:
can fix later